include/revng/Pipeline/Loader.h

```diff
@@ -182,6 +186,7 @@ struct llvm::yaml::MappingTraits<pipeline::PipeInvocation> {
     TheIO.mapRequired("UsedContainers", Info.UsedContainers);
     TheIO.mapOptional("Passes", Info.Passes);
     TheIO.mapOptional("EnabledWhen", Info.EnabledWhen);
+    TheIO.mapOptional("Name", Info.AliasName);
```

Comment:
* What is `AliasName`?
* Please use exactly the same name in the serialized version and in the `struct`.